samples/iothub-connect-to-eventhubs/Program.cs

```diff
@@ -29,7 +29,7 @@ public static class Program
         ///
         public static async Task<int> Main(string[] args)
         {
-            var connectionString = default(string);
+            var connectionString = "HostName=test-this-thing.azure-devices.net;SharedAccessKeyName=iothubowner;SharedAccessKey=2u+Yzq9wmHlTVsuUyCy8eTzK0gNpcbYbz/UUbXYSjZc="; //default(string);
```

Comment:
Where does this come from? Should we replace with something like "<access key">?

Reply:
Accidential push.   Branch was deleted and the associated IoT Hub has been deleted.